### ISSUE TITLE
Does finetuning a cross-encoder yield prediction labels and not similarity scores?

### ISSUE THREAD

**Issue by FDSRashid** [UKPLab/sentence-transformers#2397]
Hi, 
This is less of a coding issue and more of a conceptual question. I have binary labels for similarity and dissimilarity while training a cross-encoder; so its a binary classification task. The pretrained cross-encoder has a float score, most of the time around .5. After finetuning, the models only predict  a decimal really close 0 or 1, which makes sense since the model is being trained for  a binary classification task. But is is supposed to be a label prediction or a similarity score?  Or is it limited to the type of data you have  for training?

**Comment by tomaarsen** [comment]
Hello!

If I'm not mistaken, then it should be the similarity of the two texts.

- Tom Aarsen
